nbd_http_server.py

```diff
@@ -68,13 +68,19 @@ def run_or_ignore(fun):
 
 THREAD_PRINT_LOCK = threading.Lock()
 
+VERBOSE = False
+
 def thread_print(str):
     with THREAD_PRINT_LOCK:
         print(str)
 
 def eprint(str):
     thread_print(OUTPUT_PREFIX + str)
 
+def dprint(str):
+    if VERBOSE:
+        print(OUTPUT_PREFIX + str, file=sys.stderr)
```

Comment:
Same here

Reply:
fixed in [966919e](https://github.com/xcp-ng/http-nbd-transfer/pull/14/commits/966919e8f056c44b482f5e6e02a6e7d3e9d5243c)